addons/point_of_sale/static/src/js/Popups/EditListInput.js

```diff
@@ -13,7 +13,7 @@ odoo.define('point_of_sale.EditListInput', function(require) {
     }
     EditListInput.template = 'EditListInput';
 
-    Registries.Component.add(EditListInput);
+    Registries.PosComponentRegistry.add(EditListInput);
```

Comment:
Finally, don't rename this registry, it makes a lot of noize.

Reply:
You can also rename PosModelRegistry => Model

Reply:
Okay